docs/framework/docker/aspnetmvc.md

```diff
@@ -0,0 +1,241 @@
+and deploy it in a Windows container.
+
+You'll start with an existing ASP.NET MVC application. You'll see how to
+build the assets to publish using Visual Studio. You'll use Docker
+to build the image that contains your application, and
+runs that application when it is started. You'll see how to connect to your
+application running in a Windows container.
+
+If you are unfamiliar with Docker, you can learn more about the Docker
+architecture by reading the 
+[Docker Overview](https://docs.docker.com/engine/understanding-docker/)
+on the Docker site. 
+
+The application you'll run in a container is a simple website that
+answers questions randomly. This application
+is a basic MVC application with no authentication support, or database
+storage. That lets you focus on moving the web tier to a container. Later
+topics show how to move and manage persistent storage in containerized
+applications.
+
+Moving your application involves these steps:
+
+1. [Creating a publish task to build the assets for an image.](#publish-script)
+2. [Building a Docker image that will run your application.](#build-the-image)
+3. [Starting a Docker container that runs your image.](#start-a-container)
+4. [Verifying the application using your browser.](#verify-in-the-browser)
+
+The finished application is located in the
+[dotnet/core-docs repository on GitHub](https://github.com/dotnet/core-docs/tree/master/samples/framework/docker/MVCRandomAnswerGenerator).
+
+## Prerequisites
+
+Your development machine must be running Windows 10 Anniversary Update, or newe,
+or, Windows Server 2016, or newer.
+
+Before starting, you need to install [Docker for Windows](https://docs.docker.com/docker-for-windows/).
+You need to install version 1.12 Beta 26, or newer for Windows container support.
+After installing and starting Docker, you'll need to right-click on the
+tray icon and select **Switch to Windows containers...** in order to run
+Docker images based on Windows. This command takes a few seconds to
+execute:
+
+![Windows Container][windows-container]
+
+
+## Publish script
+
+You can use the Visual Studio Publish command to create a publish profile
+for your application. You'll copy all the published assets to your Docker
+image later in this tutorial. The publish command will put all the publish
+assets in one directory tree that you will copy to your target image.
+Right-click on your solution in Visual Studio, and select "Publish".
+Click the "Custom" profile button, and then select "File System" as the
+method (see image). Choose a new directory in your solution folder. (The
+downloaded sample uses "containerImage" as the folder name).
+
+![Publish Connection][publish-connection]
+
+Next, open the "File Publish Options" section of the "Settings" tab. Select
+"Precompile during publishing". This optimization means that you be
+compiling views in the Docker container, you are copying the precompiled
+views.
```

Comment:
> Next, open the "File Publish Options" section of the "Settings" tab. Select "Precompile during publishing". This optimization means that you be compiling views in the Docker container, you are copying the precompiled views.

Same thing here regarding bold instead of quotes and the bullet points.